slash/react/src/Link/Link.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+import saveIcons from "@material-symbols/svg-400/outlined/save.svg";
+import editIcons from "@material-symbols/svg-400/outlined/edit.svg";
+import { Link, type LinkProps } from "./Link";
+import { Svg } from "../Svg";
+
+const CssIcons: Record<string, string> = {
+  "(none)": "none",
+  "/public/save.svg": saveIcons,
+  "/public/edit.svg": editIcons,
+};
+
+const meta: Meta<LinkProps<"a">> = {
+  title: "Components/Link",
+  component: Link,
+  args: {
+    onClick: fn(),
+  },
+};
+
+export default meta;
+
+export const LinkStory: StoryObj<typeof Link> = {
+  name: "Link",
+  render: ({ rightIcon, leftIcon, ...args }) => {
+    const getIcon = (type: string) => {
+      if (type === "/public/save.svg") return <Svg src={saveIcons} />;
+      if (type === "/public/edit.svg") return <Svg src={editIcons} />;
```

Comment:
Meme dans une story essayons de faire du code propre:
```suggestion
      if (type === "/public/save.svg") {
        return <Svg src={saveIcons} />;
      }
      if (type === "/public/edit.svg") {
        return <Svg src={editIcons} />;
      }
```

Reply:
done